marketplace/deploy.sh

```diff
@@ -1,25 +1,38 @@
 #!/bin/bash
 
-STACK_NAME=$1
-
-TEMPLATE_BODY="file://neo4j.template.yaml"
-REGION=`aws configure get region`
+#
+# Script:  deploy.sh
+# Purpose: This script can be used to deploy the Neo4j CloudFormation template if the aws cli
+#          is correctly installed and configured
+#
 
+# User configurable variables
 Password="foo123"
-NumberOfServers="6"
+NodeCount="3"
```

Comment:
this needs to be NumberOfServers and not NodeCount since in the template we refer to NumberOfServers